eng/Version.Details.xml

```diff
@@ -19,7 +19,7 @@
     </Dependency>
     <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.ObjWriter" Version="1.0.0-alpha.1.22364.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
       <Uri>https://github.com/dotnet/llvm-project</Uri>
-      <Sha>75e182c26dbf5d9f8ccca0ac68ee63b0bebb17b8</Sha>
+      <Sha>754d13817d834b716d339183e21aac7d2489c496</Sha>
```

Comment:
@mmitche why is it updating SHA but not version?

Reply:
The sha was incorrect: https://github.com/dotnet/runtime/blob/f52d8c59bb49360eb2cbeeb863c5856ebd62adda/eng/Version.Details.xml#L33